tests/countries/test_singapore.py

```diff
@@ -26,138 +24,159 @@ def test_country_aliases(self):
     def test_common(self):
         self.assertNonObservedHolidayName("New Year's Day", "2022-01-01")
 
-    def test_Singapore(self):
-        # <= 1968 holidays
-        self.assertIn(date(1968, 4, 13), self.holidays)
-        self.assertIn(date(1968, 4, 15), self.holidays)
-        self.assertIn(date(1968, 12, 26), self.holidays)
-        # latest polling day
-        self.assertIn(date(2015, 9, 11), self.holidays)
-        # Year with lunar leap month
-        self.assertIn(date(2015, 8, 7), self.holidays)
-        # holidays estimated using lunar calendar
-        self.assertIn(date(2050, 6, 4), self.holidays)  # Vesak Day
-        self.assertIn(date(2050, 11, 12), self.holidays)  # Deepavali
+    def test_singapore(self):
```

Comment:
This test's name is too broad and it contains a bunch of unrelated dates. Just a note to address this later.